setup/functions/auth.js

```diff
@@ -25,7 +25,7 @@ export default (request, response) => {
         return response.send();
     };
 
-    let reject = () => {
```

Comment:
renaming this to `unauthorized` because `Promise` convention is to name params `resolve, reject`